docs/ui.md

```diff
@@ -1,20 +1,40 @@
-# Kubernetes UI Instructions
+# Kubernetes User Interface
+Kubernetes has a web-based user interface that displays the current cluster state graphically. It is accessible via `https://<kubernetes-master>/ui`, which redirects to `https://<kubernetes-master>/api/v1/proxy/namespaces/kube-system/services/kube-ui/#/dashboard/`.
 
-## Kubernetes User Interface
-Kubernetes has an extensible user interface with default functionality that describes the current cluster. See the [README](../www/README.md) in the www directory for more information.
-
-### Running locally
-Assuming that you have a cluster running locally at `localhost:8080`, as described [here](getting-started-guides/locally.md), you can run the UI against it with kubectl:
+## Accessing the UI
+By default, the Kubernetes UI is deployed remotely as a cluster addon. To access it, visit `https://<kubernetes-master>/ui`, which redirects to `https://<kubernetes-master>/api/v1/proxy/namespaces/kube-system/services/kube-ui/#/dashboard/`.
 
+If the [`kube-addons.sh`](../cluster/saltbase/salt/kube-addons/kube-addons.sh) script is not running, the kube-ui service will not be started. In this case, it can be started manually with:
```

Comment:
I would re-phrase this as something like "If you find that you're not able to access the UI, it may be because the kube-ui service has not been started on your cluster. In that case you can start it manually with with [...commands...] Normally, this should be taken care of automatically by the [`kube-addons.sh`](../cluster/saltbase/salt/kube-addons/kube-addons.sh) script that runs on the master"